src/levels.rs

```diff
@@ -1,13 +1,16 @@
 use std::ffi::c_uint;
 use std::ops::Neg;
-use strum::EnumCount;
-
-pub type ObuMetaType = c_uint;
-pub const OBU_META_TIMECODE: ObuMetaType = 5;
-pub const OBU_META_ITUT_T35: ObuMetaType = 4;
-pub const OBU_META_SCALABILITY: ObuMetaType = 3;
-pub const OBU_META_HDR_MDCV: ObuMetaType = 2;
-pub const OBU_META_HDR_CLL: ObuMetaType = 1;
+use strum::{EnumCount, FromRepr};
```

Comment:
Could you put one import per line?  I've been doing that with these rust-analyzer settings in my `.vscode/settings.json`:

```json
    "rust-analyzer.imports.granularity.group": "item",
    "rust-analyzer.imports.prefix": "crate",
```